test/puppetlabs/puppetdb/command/dlo_test.clj

```diff
@@ -54,11 +93,23 @@
                #(err-attempt-line? 1 %)
                #(= "java.lang.Exception: thud-1" %)]
               (-> (:info discards) .toFile io/reader line-seq)))
-         (is (= content (-> (:command discards) .toFile io/reader slurp))))
-       (is (= 1 (cval ["global" "messages"])))
-       (is (= cmd-size (cval ["global" "filesize"])))
-       (is (= 1 (cval ["replace catalog" "messages"])))
-       (is (= cmd-size (cval ["replace catalog" "filesize"])))))))
+         (is (Arrays/equals content
```

Comment:
Is there a reason not to compare the content as strings? We have functions that make this easy and if they are different, seems like it'd be easier to find the issue in a string vs. byte array.